### PR TITLE
Pulumi v3.81.0

### PR DESCRIPTION
**Pull request by t0yv0**
Dependencies: pulumi/{sdk,pkg} v3.81.0

Fixes https://github.com/pulumi/pulumi-terraform-bridge/issues/1356

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -63,7 +63,18 @@ require (
 	github.com/BurntSushi/toml v1.2.1 // indirect
 	github.com/Masterminds/semver/v3 v3.1.1 // indirect
 	github.com/Masterminds/sprig/v3 v3.2.2 // indirect
+	github.com/atotto/clipboard v0.1.4 // indirect
+	github.com/aymanbagabas/go-osc52/v2 v2.0.1 // indirect
+	github.com/charmbracelet/bubbles v0.16.1 // indirect
```
Comment (t0yv0) [inline go.mod:68]: Very interesting

We keep growing the dependency set.

**Comment by t0yv0** [comment]
I'm surprised downstream tests mention pulumix. Something is off if downstream starts depending on pulumix but the bridge does not yet. 

```
Error: go/azuread/accessPackage.go:13:2: no required module provides package github.com/pulumi/pulumi/sdk/v3/go/pulumix; to add it:
	go get github.com/pulumi/pulumi/sdk/v3/go/pulumix
Error: go/azuread/accessPackage.go:13:2: no required module provides package github.com/pulumi/pulumi/sdk/v3/go/pulumix; to add it:
	go get github.com/pulumi/pulumi/sdk/v3/go/pulumix
make: *** [Makefile:43: build_go] Error 123
```

**Comment by iwahbe** [comment]
> I'm surprised downstream tests mention pulumix. Something is off if downstream starts depending on pulumix but the bridge does not yet.
> 
> ```
> Error: go/azuread/accessPackage.go:13:2: no required module provides package github.com/pulumi/pulumi/sdk/v3/go/pulumix; to add it:
> 	go get github.com/pulumi/pulumi/sdk/v3/go/pulumix
> Error: go/azuread/accessPackage.go:13:2: no required module provides package github.com/pulumi/pulumi/sdk/v3/go/pulumix; to add it:
> 	go get github.com/pulumi/pulumi/sdk/v3/go/pulumix
> make: *** [Makefile:43: build_go] Error 123
> ```

The bridge doesn't depend on `pulumix`, but the sdks it generates do.

**Review by iwahbe — APPROVED** [review]

**Comment by t0yv0** [comment]
This is a little suspect, I'd like to double check tomorrow. So in that case should the bridge bump the dependency to such a version of pulumi/sdk as would include pulumix so that it propagates automatically? Or maybe we can call out in release notes that this upgrade necessitates adding x.y.z pulumi/sdk dependency.

**Comment by iwahbe** [comment]
> This is a little suspect, I'd like to double check tomorrow. So in that case should the bridge bump the dependency to such a version of pulumi/sdk as would include pulumix so that it propagates automatically? Or maybe we can call out in release notes that this upgrade necessitates adding x.y.z pulumi/sdk dependency.

Let's try adding a `go mod tidy` step to our upgrade process. I believe that would resolve this.

**Comment by t0yv0** [comment]
So this is interesting. https://github.com/pulumi/pulumi-random/pull/402 fixed it for random. https://github.com/pulumi/pulumi-azuread/pull/451 should fix it for azuread.

**Comment by t0yv0** [comment]
https://github.com/pulumi/action-test-provider-downstream/issues/19 logged but it seems rare enough that can punt for now.

**Pull request merged**